src/styles/utils/_variables.scss

```diff
@@ -1,2 +1,5 @@
 $acdc-color: #00acdc;
 $black-color: #060b35;
+$star-height: 16px;
+$star-width: 16px;
```

Comment:
just `$star-size: 16px;`. check in other places for that problem

Reply:
sry. didnt think about it. a little bit late in my time zone. falling asleep)))